.ci/updatecli/updatecli-bump-golang.yml

```diff
@@ -124,21 +112,12 @@ targets:
         - Dockerfile
         - Dockerfile.skaffold
       matchpattern: 'ARG GO_VERSION=\d+.\d+.\d+'
-  update-gomod-minor-version:
-    name: "Update go.mod minor version"
-    sourceid: gomod
-    scmid: githubConfig
-    kind: file
-    spec:
-      content: 'go {{ source "gomod" }}'
-      file: go.mod
-      matchpattern: 'go \d+.\d+'
-  update-gomod-toolchain-version:
```

Comment:
we don't define a toolchain in Elastic Agent's `go.mod`, this fails the pipeline.